ui/components/multichain/import-tokens-modal/import-tokens-modal.js

```diff
@@ -409,44 +439,63 @@ export const ImportTokensModal = ({ onClose }) => {
   return (
     <Modal
       isOpen
+      isClosedOnOutsideClick={false}
       onClose={() => {
-        dispatch(clearPendingTokens());
-        onClose();
+        if (
+          Object.keys(selectedTokens).length === 0 ||
+          Object.keys(pendingTokens).length === 0
+        ) {
+          dispatch(clearPendingTokens());
+          onClose();
+        } else {
+          dispatch(showImportTokensExitModal());
+        }
       }}
-      className="import-tokens-modal"
+      className="import-tokens-modal scrollable"
     >
       <ModalOverlay />
-      <ModalContent>
+      <ModalContent
+        className="contentClass"
+        modalDialogProps={{ className: 'contentDialogClass' }}
+      >
         <ModalHeader
           onBack={isConfirming ? () => setMode('') : null}
+          paddingBottom={4}
+          paddingRight={4}
+          paddingLeft={4}
           onClose={() => {
-            dispatch(clearPendingTokens());
-            onClose();
+            if (Object.keys(selectedTokens).length === 0) {
+              dispatch(clearPendingTokens());
+              onClose();
+            } else {
+              dispatch(showImportTokensExitModal());
+            }
           }}
         >
           {t('importTokensCamelCase')}
         </ModalHeader>
-        <Box marginTop={6}>
-          {isConfirming ? (
-            <ImportTokensModalConfirm
-              onBackClick={() => {
-                dispatch(clearPendingTokens());
-                setMode('');
-              }}
-              onImportClick={async () => {
-                await handleAddTokens();
-                onClose();
-              }}
-            />
-          ) : (
-            <>
-              <Tabs t={t}>
-                {showSearchTab ? (
-                  <Tab tabKey="search" name={t('search')}>
-                    <Box paddingTop={4} paddingBottom={4}>
-                      {useTokenDetection ? null : (
-                        <BannerAlert severity={Severity.Info} marginBottom={4}>
-                          <Text>
+        <Box>
+          <Tabs t={t} tabsClassName="import-tokens-tabs">
+            {showSearchTab ? (
+              <Tab
+                activeClassName="import-tokens-modal__active-tab"
+                buttonClassName="import-tokens-modal__button-tab"
+                tabKey="search"
+                name={t('search')}
+              >
+                {isConfirming ? (
+                  <ImportTokensModalConfirm />
+                ) : (
+                  <Box paddingTop={4}>
+                    {useTokenDetection ? null : (
+                      <Box paddingLeft={4} paddingRight={4}>
+                        <BannerAlert
+                          severity={Severity.Info}
+                          marginBottom={4}
+                          paddingLeft={4}
+                          paddingRight={4}
+                        >
+                          <Text variant={TextVariant.bodyMd} fontSize="16">
```

Comment:
Should `fontSize="16"` be there?  I think we should try to stick to variants only cc: @georgewrmarshall